runtime/index.d.ts

```diff
@@ -28,7 +28,7 @@ declare module '@sapper/server' {
 		ignore?: Ignore
 	}
 
-	export function middleware(opts?: MiddlewareOptions): Handler;
+	export function middleware<Session = unknown>(opts?: MiddlewareOptions<Session>): Handler;
```

Comment:
I don't think this works correctly unless we also parameterize `MiddlewareOptions`